packages/@aws-cdk/aws-cloudfront/test/private/cache-behavior.test.ts

```diff
@@ -4,25 +4,23 @@ import { AllowedMethods, HttpOrigin } from '../../lib';
 import { CacheBehavior } from '../../lib/private/cache-behavior';
 
 let app: App;
-let stack: Stack;
 
 beforeEach(() => {
   app = new App();
-  stack = new Stack(app, 'Stack', {
+  new Stack(app, 'Stack', {
     env: { account: '1234', region: 'testregion' },
   });
 });
 
 test('renders the minimum template with an origin and path specified', () => {
   const origin = new HttpOrigin('www.example.com');
-  const behavior = new CacheBehavior({
+  const behavior = new CacheBehavior('origin_id', {
     origin,
```

Comment:
As I said above, this feels a little weird. It's just for the sake of the tests, but worth considering... does this class really need the origin at this point?

Reply:
Done!